lib/Auth/Process/PerunEntitlement.php

```diff
@@ -139,6 +140,16 @@ private function getEduPersonEntitlement(&$request)
     private function getForwardedEduPersonEntitlement(&$request)
     {
         $forwardedEduPersonEntitlement = [];
+
+        if (!isset($request['perun']['user'])) {
+            Logger::error(
+                'perun:PerunEntitlement: ' .
+                'missing mandatory field \'perun.user\' in request.' .
+                'Hint: Did you configure PerunIdentity filter before this filter?'
```

Comment:
I would change the hint to: _"Did you configure PerunIdentity filter to run before this filer?_